maintenance_curation.es.Rmd

```diff
@@ -1,217 +1,116 @@
 # Política de conservación de paquetes {#curationpolicy}
 
-```{block, type="summaryblock"}
-Este capítulo resume una política de conservación propuesta para el mantenimiento continuo de rOpenSci
-mantenimiento continuo de los paquetes desarrollados como parte de las actividades de rOpenSci
-y/o bajo la organización rOpenSci GitHub. Esta política de conservación pretende
-apoyar estos objetivos:
+\`\`\`{block, type='summaryblock'} Este capítulo resume una propuesta de política de conservación para el mantenimiento continuo por parte de rOpenSci de los paquetes desarrollados como parte de las actividades de rOpenSci y/o bajo la organización rOpenSci GitHub. Esta política de conservación pretende apoyar estos objetivos:
```

Comment:
This is very weird. In every translation deepl also changes `"summaryblock"` to `'summaryblock'`. Perhaps this can be added to the glossary? (Kind of a hack, perhaps)

Reply:
I think it's more tinkr fault. :thinking: